include/boost/redis/connection.hpp

```diff
@@ -412,24 +416,46 @@ inline system::error_code check_config(const config& cfg)
    return system::error_code{};
 }
 
+inline system::error_code translate_parallel_group_errors(
```

Comment:
I know `inline` functions might result in better performance but for this specific case it won't make a difference. I want to make compilation as fast as possible, so I prefer to move this to an `.ipp` file.